CHANGES.rst

```diff
@@ -13,6 +13,9 @@ Deprecated
 Features added
 --------------
 
+* #12448: Add ``--remove-old`` option to ``sphinx-apidoc``.
```

Comment:
Could you perhaps add a documentation to the `sphinx-apidoc` page (do we have one?). Also I prefer to not use the `-r` as for `--remove-old` because `-r` seems more relevant for a 'recursive' option instead. I think a long option is clearer here.

Reply:
done 👍

Reply:
Ok, there's a comment that I disappeared when I reviewed. I wanted to ask if you could add a link to the option (so that readers can click on it and see what it does).

Reply:
Oh shoot, saw it after I merged 😅, I'll add it in the next PR